tests/model/ArrayListTest.php

```diff
@@ -424,6 +424,31 @@ public function testMultiSort() {
 		$this->assertEquals($list->first()->ID, 1, 'Aron.2 should be first in the list');
 		$this->assertEquals($list->last()->ID, 3, 'Bert.3 should be last in the list');
 	}
+
+	/**
+	 * Check that we don't cause recursion errors with array_multisort() and circular dependencies
+	 */
+	public function testSortWithCircularDependencies() {
+		$items = ArrayList::create();
```

Comment:
Do we want to be using Injector here, to create the ArrayList, or do we really want a real `ArrayList`?